eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,7 +1,7 @@
 param (
   [string]$TargetDirectory = "", # Code path to code owners. e.g sdk/core/azure-amqp
   [string]$CodeOwnerFileLocation = (Resolve-Path $PSScriptRoot/../../../.github/CODEOWNERS), # The absolute path of CODEOWNERS file. 
-  [string]$ToolVersion = "1.0.0-dev.20221215.1", 
+  [string]$ToolVersion = "1.0.0-dev.20230108.6", 
```

Comment:
I remember there was a discussion on creating a PR once tools get published in pipeline. 
I can pick up the item once available. It reminds us to update the tool version.